src/main/resources/templates/beverage/index.html

```diff
@@ -10,11 +10,13 @@ <h2 th:unless="${beverages} and ${beverages.size()}">We're sorry, there are curr
 	<table class="table table-striped" th:if="${beverages} and ${beverages.size()} > 0">
         <thead>
         <tr>
-            <th>Beverage Name</th>
+            <th>Beverage Name<td>Edit</td><td>Copy</td></th>
         </tr>
         </thead>
         <tr th:each="beverage : ${beverages}">
             <td><a th:text="${beverage.name}" th:value = "${beverage.id}"th:href="'/recipes?beverageId=' + ${beverage.id}"></a></td>
+            <td><a th:text="Edit" th:value = "${beverage.id}" th:href="'beverages/edit/' + ${beverage.id}"></a></td>
+            <td><a th:text="Copy" th:value = "${beverage.id}" th:href="'beverages/copy/' + ${beverage.id}"></a></td>
         </tr>
     </table>
 </div>
```

Comment:
Love the way you have the edit and copy functionality separated here for the beverages. Just a heads-up, it is not currently set up this way for categories. You still have the edit/copy functionality contained under a single view. I am not sure if that's intentional or unintended in the editing process, but I think we should mirror it there as well!